common/static/js/vendor/draggabilly.pkgd.js

```diff
@@ -1748,7 +1748,9 @@ return Unidragger;
 
   if ( typeof define == 'function' && define.amd ) {
     // AMD
-    define( [
+    define(
```

Comment:
Please add a comment to the code explaining why this is needed and that it is specifically for Studio (since LMS uses a namespaced version).